apps/system/test/marionette/lockscreen_unlock_to_homescreen_test.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+var dirapps = require('path').resolve(__dirname + '/../../../');
+var Settings = require(dirapps + '/settings/test/marionette/app/app'),
+    LockScreen = require('./lib/lockscreen'),
+    LockScreenPasscodeUnlockActions = require(
+      './lib/lockscreen_passcode_unlock_actions'),
+    Promise = require('es6-promise').Promise, // jshint ignore:line
```

Comment:
Well its an **intermittent** error only happens on TaskCluster. From my previous 30 times test result, it seemed that environments on **some** instances lack of native Promise, which was supposed to be either failed on all instances or passed without any problem. So the best way to work this around without digging into the TaskCluster infrastructure is to require it explicitly.